build/attestations/provenance.md

```diff
@@ -26,6 +26,9 @@ $ docker buildx build --tag <namespace>/<image>:<version> \
     --attest type=provenance,mode=[min,max] .
 ```
 
+Alternatively, you can use the shorthand `--provenance` option instead of `--attest type=provenance`.
+Note that the shorthand version doesn't allow you to specify the `mode` configuration parameter.
```

Comment:
Aha, it does! :tada: (sorry, my miscommunication I think).

`--provenance=true/false/csv`, so `--provenance=mode=max` is perfectly allowed.

Reply:
OK! I actually think this was a stab in the dark on my part. Meant to test it out but I guess I never got to it. Will fix!